javascript-primitives-and-variables/main.js

```diff
@@ -1,15 +1,15 @@
-let fullName = 'Lawrence Wang'
-let isCool = true
-let totalPets = 0
+const fullName = 'Lawrence Wang'
+const isCool = true
+const totalPets = 0
 console.log('value of fullName: ',  fullName);
 console.log('value of of isCool: ',  isCool);
 console.log('value of totalPets: ',  totalPets);
 console.log('typeof fullName: ', typeof fullName)
 console.log('typeof isCool: ', typeof isCool);
 console.log('typeof totalPets: ', typeof totalPets);
-let unknown = null
+const unknown = null
 console.log('value of unknown: ', unknown)
 console.log('typeof unknown: ', typeof unknown);
-let never
+const never
```

Comment:
This should still be using `let` actually, because we haven't given it a value yet. So in order to eventually use this variable, we will need to give it a value at some point, so it would need to use `let`